alica_tests/src/test/test_task_assignment.cpp

```diff
@@ -93,6 +93,8 @@ TEST_F(TaskAssignmentTest, constructTaskAssignment)
     bool ok = ps->getPlansForState(rp, inputWrappers, robots, o_plans);
     EXPECT_TRUE(ok);
     EXPECT_EQ(o_plans.size(), 1u);
+    delete wrapper;
```

Comment:
I would prefer we use unique_ptr over raw pointers when appropriate rather than explicitly deleting